Cargo.toml

```diff
@@ -36,7 +37,7 @@ async-h1 = { version = "2.0.1", optional = true }
 async-sse = "3.0.0"
 async-std = { version = "1.6.0", features = ["unstable"] }
 femme = "2.0.1"
-http-types = "2.2.1"
+http-types = { version = "2.2.1", default-features = false }
```

Comment:
What is the purpose of this?

Reply:
So that applications that do not need signed or encrypted cookies don't have to needlessly pull in a bunch of crypto crates.